src/brain/gocdSlackFeeds/index.ts

```diff
@@ -366,10 +368,36 @@ const goCDCustomJobRunnerFeed = new DeployFeed({
   channelID: FEED_GOCD_JOB_RUNNER_CHANNEL_ID,
   msgType: SlackMessage.GOCD_CUSTOM_JOB_RUN,
   pipelineFilter: (pipeline) => {
-    // We only want to capture updates for the GoCD Job Runner pipeline,
-    // whether successful or failed
+    // We only want to capture updates for the GoCD Job Runner pipeline
     return pipeline.name === GOCD_CUSTOM_JOB_PIPELINE_NAME;
   },
+  replyCallback: async (pipeline) => {
+    if (pipeline.stage.result.toLowerCase() === 'unknown') {
+      return [];
+    }
+
+    const blocks = [
+      header(
+        plaintext(`${pipeline.name} stage update`)
+      ),
+      stageBlock(pipeline),
+    ];
+
+    const approvedBy = pipeline.stage['approved-by'];
```

Comment:
@IanWoodard does this `approved-by` actually contain the user who kicked off the deployment on the stages in the pipeline after the first one? In the GoCD UI I see "Automatically triggered" instead of "Triggered by <user>".

Not the worst case if we don't have the user to @ in the reply, since they're already tagged in the channel message, so replying in thread at all should re-notify (unless they have this setting turned off?). Safer to @ so would prefer that if possible.

Reply:
I'd add some tests for the edge cases here, but it should work for most cases (not the case where it is auto-deployed)